### PR TITLE
Let Paket manage FSharp.Core

### PR DESCRIPTION
**Pull request by simendsjo**
* Remove FSharp.Core direct references from fsproj files
* Add FSharp.Core nuget reference and force redirects
* Add App.config so paket can add redirects

Note that Paket has automatically rearranged a bit in the project files and removed the patch version numbers from the lock file.

### CODE REVIEW

**Comment by simendsjo** [comment]
Refs #249

File: src/FSharp.ProjectTemplate/App.config [hunk 1]
```diff
@@ -0,0 +1,4 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>  
+
```
Comment (forki) [inline src/FSharp.ProjectTemplate/App.config]: Fsharp.core is missing. Did you run "paket install --hard"?

Comment (forki) [inline src/FSharp.ProjectTemplate/App.config]: Fsharp.core is missing. Did you run "paket install --hard"?

Comment (forki) [inline src/FSharp.ProjectTemplate/App.config]: Ah I think you forgot to add fsharp.core to the 2 paket.references

**Comment by simendsjo** [comment]
Damn.. Trying to use Git from Windows when I only installed the Github for Windows is a realy PITA! No idea what Github means by sync, but I got some really odd behavior.

File: tests/FSharp.ProjectTemplate.Tests/FSharp.ProjectTemplate.Tests.fsproj [hunk 2]
```diff
@@ -54,39 +54,35 @@
...
     </ProjectReference>
   </ItemGroup>
-  <!-- To modify your build process, add your task inside one of the targets below and uncomment it. 
-       Other similar extension points exist, see Microsoft.Common.targets.
-  <Target Name="BeforeBuild">
-  </Target>
-  <Target Name="AfterBuild">
-  </Target>
-  -->
-  <Import Project="..\..\.paket\paket.targets" />
```
Comment (dsyme) [inline tests/FSharp.ProjectTemplate.Tests/FSharp.ProjectTemplate.Tests.fsproj]: Why is the reference to paket.targets removed?  thanks

Comment (cdrnet) [inline tests/FSharp.ProjectTemplate.Tests/FSharp.ProjectTemplate.Tests.fsproj]: Seems it just moved up to L64

Comment (simendsjo) [inline tests/FSharp.ProjectTemplate.Tests/FSharp.ProjectTemplate.Tests.fsproj]: Sometimes running a new version of Paket reorders things in the project files. I could move this to a commit of its own if you'd like.

**Comment by forki** [comment]
thanks

**Pull request merged**